resources/js/Pages/Dashboard.vue

```diff
@@ -1,22 +1,122 @@
-            <h2 class="font-semibold text-xl text-gray-800 leading-tight">Dashboard</h2>
+            <h2 class="text-center">Dashboard</h2>
         </template>
-
-        <div class="py-12">
-            <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+        <div class="container">
+            <div class="row">
                 <div class="bg-white overflow-hidden shadow-sm sm:rounded-lg">
-                    <div class="p-6 text-gray-900">You're logged in!</div>
+                    <div class="text-center">You're logged in!</div>
+                    <form @submit.prevent="submitPost" class="new-post row g-3">
+                        <InputLabel for="new-post" value="Write here" />
+                        <div class="col-md-8">
+                            <TextInput
+                            id="new-post"
+                            type="text"
+                            placeholder="write something here"
+                            v-model="form.post_content"
+                            autofocus
+                            />
+                        </div>
+                        <PrimaryButton class="col-md-4">
+                            Post
+                        </PrimaryButton>
+                    </form>
+                    <div class="post_items row g-3">
+                        <div class="post_item" v-for="(post, index) in posts">
+                            <div class="post_user">Charles</div>
+                            <span class="post_date">{{ post.created_at}}</span>
+                            <p class="post_content">{{ post.post_content}}</p>
+                            <span class="post_likes">
+                                <i class="zmdi zmdi-thumb-up"></i>Likes
+                            </span>
+                            <span class="post_comments_count" @click="showComments">
+                                <i class="zmdi zmdi-comment-text"></i>Comments
+                            </span>
+                            <div class="post_comments">
+                                <p class="post_comment">
+                                    <span class="d-block"><strong>ABC</strong></span>
+                                    <span class="d-block">Hello testing comment</span>                                
+                                </p>
+                                <p class="post_comment">
+                                    <span class="d-block"><strong>XYZ</strong></span>
+                                    <span class="d-block">Hello testing comment12</span>                                
+                                </p>
+                                <p class="post_comment">
+                                    <span class="d-block"><strong>PQR</strong></span>
+                                    <span class="d-block">Hello testing comment123</span>                                
+                                </p>
+                            </div>
```

Comment:
I gather that you're still working on this part of the feature - I tested this and my comment was saved to the DB, but only these test comments display. We should make this dynamic.